### PR TITLE
Add support for halium 7++ (both hybris and hidl)

### PR DESCRIPTION
**Pull request by mariogrip**
This is rebased on latest xenial and removed commit that disabled the tests, this should now pass tests.

I also cleaned up the cmake files to allow to build with and without hybris. 

Huge thanks to @erfanoabdi

### CODE REVIEW

File: include/biometry/hardware/biometry.h [hunk 1]
```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright © 2020 UBports foundation Ltd.
```
Comment (mariogrip) [inline include/biometry/hardware/biometry.h]: We are not Ltd, will fix that

**Comment by Flohack74** [comment]
Did you see https://github.com/ubports/biometryd/pull/3/commits/ec83f6bd23c80ee45fc91816ec657b6c007ba7ad#diff-ffa9aa33fbe5bc1377bbf42a97969bae281df0ac5ab0836ae97c87d6f36d1d1aR129 - I have at least on angler the issue that enrollment gets stuck all the time and as discussed with @erfanoabdi we need to disable the completion there. I just dont know what that means in the end, but enrollment was working thereafter.

**Comment by erfanoabdi** [comment]
not sure disabling `postEnroll` hurts on other devices or not but we can leave it disabled
if any device needed the post enroll function in future by adding a prop check we can handle it..

**Comment by peat-psuwit** [comment]
Migrated to GitLab.

https://gitlab.com/ubports/core/biometryd/-/merge_requests/5

**Pull request closed without merging**